scanapp/tests/test_models.py

```diff
@@ -0,0 +1,260 @@
+from scanapp.models import Package
+from scanapp.models import Scan
+from scanapp.models import ScanError
+from scanapp.models import ScannedFile
+
+
+class ScanTestCase(TestCase):
+    def test_scan_added(self):
+        user = User.objects.create_user(username='username', password='password')
+        scan = Scan.objects.create(
+            user=user,
+            url='https;//github.com',
+            scan_directory='media/username',
+            scancode_notice='Some dummy notice',
+            scancode_version='2.0.0rc3',
+            files_count=200,
+            scan_start_time=timezone.now(),
+            scan_end_time=timezone.now()
+        )
+        
+        self.assertEqual('2.0.0rc3', scan.scancode_version)
+        self.assertEqual(str(scan.url), str(scan))
+        self.assertEqual('scans', str(scan._meta.verbose_name_plural))
+
+
+class ScannedFileTestCase(TestCase):
+    def test_scanned_file_added(self):
+        user = User.objects.create_user(username='username', password='password')
+        scan = Scan.objects.create(
+            user=user,
+            url='https;//github.com',
+            scan_directory='media/username',
+            scancode_notice='Some dummy notice',
+            scancode_version='2.0.0rc3',
+            files_count=200,
+            scan_start_time=timezone.now(),
+            scan_end_time=timezone.now()
+        )
+        scanned_file = ScannedFile.objects.create(scan=scan, path='/home/nexb/server/')
+
+        self.assertEqual('/home/nexb/server/', scanned_file.path)
+        self.assertEqual(scanned_file.path, str(scanned_file))
+        self.assertEqual('scanned files', str(scanned_file._meta.verbose_name_plural))
+
+
+class LicenseTestCase(TestCase):
+    def test_license_added(self):
+        user = User.objects.create_user(username='username', password='password')
+        scan = Scan.objects.create(
+            user=user,
+            url='https;//github.com',
+            scan_directory='media/username',
+            scancode_notice='Some dummy notice',
+            scancode_version='2.0.0rc3',
+            files_count=200,
+            scan_start_time=timezone.now(),
+            scan_end_time=timezone.now()
+        )
+        scanned_file = ScannedFile.objects.create(scan=scan, path='/home/nexb/server/')
+        license = License.objects.create(
+            scanned_file=scanned_file, 
+            key='A',
+            score=78,
+            short_name='mit',
+            category='some category',
+            owner='mit',
+            homepage_url='https://github.com/',
+            text_url='http://github.com/',
+            dejacode_url='https://github.com',
+            spdx_license_key='mit', 
+            spdx_url='https://github.com/',
+            start_line=21,
+            end_line=567,
+            matched_rule=json.loads('{"url": ["https://github.com", "https://google.com"]}')
+        )
+        self.assertEqual('mit', license.owner)
```

Comment:
You need to get the object you have saved ... otherwise you are not testing anything